remote-api/src/main/java/org/alfresco/rest/api/impl/DeletedNodesImpl.java

```diff
@@ -244,4 +246,21 @@ public Rendition getRendition(String archivedId, String renditionId, Parameters
         NodeRef nodeRef = new NodeRef(StoreRef.STORE_REF_ARCHIVE_SPACESSTORE, archivedId);
         return renditions.getRenditions(nodeRef, parameters);
     }
+
+
+    /**
+     * @author Mikołaj Brzeziński
+     * Gets a presigned URL to directly access content.
+     * @param originalNodeId The node id for which to obtain the direct access {@code URL}
+     * @param attachment {@code true} if an attachment {@code URL} is requested, {@code false} for an embedded {@code URL}, {@code true} by default.
+     * @param validFor The time at which the direct access {@code URL} will expire.
+     * @return A direct access {@code URL} object for the content.
+     */
```

Comment:
you can replace these javadoc comments with:
```
    /**
     * {@inheritDoc}
     */
```
which will inherit the javadoc comments from the interface, removing the need to update in both places if/when the comments change :)